platform/spire/src/command.py

```diff
@@ -43,18 +43,21 @@ def configure(self, command: list, parser: argparse.ArgumentParser):
         subparsers = parser.add_subparsers()
 
         for component, subcommand in self.mapping.items():
-            doc = inspect.getdoc(subcommand)
+            doc = subcommand.short_doc()
             short_doc = doc.split('\n')[0] if doc else None
```

Comment:
Are these local variables used for anything anymore?

Reply:
oops

Reply:
on the bright side - pyflakes caught this in CI :)